### PR TITLE
Enhanced header visibility and improved student section layout

### LINKED ISSUE #76: UI Improvement for Student Section and Header Visibility Issue
The UI in the Student Section needs refinement as it doesn't appear polished. Additionally, the header becomes invisible when navigating through sections with a white background. Please assign this issue to me under the labels gssoc ext and Hacktoberfest. 
Thank you!

Screenshot (391)
Screenshot (393)

### PR DESCRIPTION
**Pull request by Shweta-1902**
## 📋 Description

This pull request enhances the visibility of the header to prevent text from fading, ensuring a better user experience. Additionally, I've made several improvements to the student section layout, including better use of background colors and more detailed content for each student.

## 🔨 Changes Made

Improved header visibility to enhance text clarity and readability.
Enhanced the student section layout to ensure it occupies the entire box.
Increased the content for each student profile to fill the designated area effectively.
Updated images for student profiles with valid links.
Adjusted the overall styling for a more cohesive and appealing appearance.

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

 I have followed all the guidelines mentioned in CONTRIBUTING.md.
 My code follows the style guidelines of this project.
 I have performed a self-review of my own code.
 I have commented my code, particularly in hard-to-understand areas.
 I have made corresponding changes to the documentation (if applicable).
 I have tested it locally and it works fine.
 Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

 UI enhancement (non-breaking change which enhances UI) 🎨

- Fixes #76 

## 📸 Screenshots (if applicable)

Screenshot (414)
Screenshot (415)
Screenshot (416)
Screenshot (419)
Screenshot (418)
Screenshot (420)
Screenshot (422)

*Thank you for contributing to Research Nexus! We look forward to reviewing your PR and getting it merged!*

### CODE REVIEW

**Comment by Shweta-1902** [comment]
changes done

File: public/index.html [hunk 1]
```diff
@@ -461,5 +488,19 @@ <h4 id="custom-follow-heading">Follow Us</h4>
...
+
+    <script>
+        // Adding background color to the header on scroll
+        window.onscroll = function() {
+            var navbar = document.querySelector(".navbar");
+            if (window.scrollY > 50) {
+                navbar.style.backgroundColor = "rgba(0, 0, 0, 0.8)"; // Dark background on scroll
+                navbar.querySelectorAll("a").forEach(a => a.style.color = "white"); // Change link color
+            } else {
+                navbar.style.backgroundColor = "rgba(0, 0, 0, 0.8)"; // Revert back to light background
```
Comment (Harshdev098) [inline public/index.html:500]: make the backgroundColor none, it will be good

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

**Comment by Shweta-1902** [comment]
done but again the text almost gets invisible on white background

**Comment by Harshdev098** [comment]
I have requested to change only the else condition backgroundColor to be none, you have made backgroundColor none to both the conditions

**Comment by Harshdev098** [comment]
Just wait a minute, will give a solution to it

**Comment by Harshdev098** [comment]
Just make the backgroundColor to be transparent in the else condition, it will work @Shweta-1902

**Comment by Shweta-1902** [comment]
done

**Comment by Harshdev098** [comment]
Can you please your PR description, because it referencing this pr to the other PR but that should not be the case it should refer to the issue you have solved `fixes #<issue_number>`.
Will merge it

**Comment by Shweta-1902** [comment]
The leaderboard didn't got updated, it shows the same score as before

**Comment by Harshdev098** [comment]
It will be grt updated soon 
I have added the labels on the PR

**Pull request merged**